packages/cli/src/commands/hydrogen/check.ts

```diff
@@ -8,14 +8,17 @@ import {
 } from '../../utils/missing-routes.js';
 
 export default class GenerateRoute extends Command {
+  static description =
+    'Returns diagnostic information about a Hydrogen storefront.';
+
   static flags = {
     path: commonFlags.path,
   };
 
   static args = [
     {
       name: 'resource',
-      description: `The resource to check. Currently only 'routes' is supported.`,
+      description: `The resource to check. Currently only 'routes' is supported`,
```

Comment:
I'd keep the period because you have two complete sentences here back to back. Here and anywhere else it applies.